src/schemas/json/golangci-lint.json

```diff
@@ -466,6 +466,7 @@
     "output": {
       "description": "Output configuration options.",
       "type": "object",
+      "additionalProperties": false,
```

Comment:
can this option be included to all high level properties? `run`, `linters`, `issues`, etc

Reply:
some sections can have extra fields related to plugins, so it's better to not be strict on all.